app-crypt/stegsnow/stegsnow-20130616.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
```

Comment:
That's a possibility, yeah. Dropping the subject is a bit colloquial, I guess; it is often found in informal documentation, though. If you think the second version is more appropriate, I'll change it right away.